modules/core/src/scenegraph/gltf-instantiator/gltf-instantiator.js

```diff
@@ -1,13 +1,13 @@
-import {Buffer, Texture2D, Model, Group} from 'luma.gl';
+import Buffer from '../../webgl/buffer';
+import Group from '../group';
+import Model from '../model';
 
 // GLTF instantiator for luma.gl
 // Walks the parsed and resolved glTF structure and builds a luma.gl scenegraph
 export default class GLTFInstantiator {
-  constructor(gl) {
+  constructor(gl, options = {}) {
     this.gl = gl;
-    this.options = {
-      removeEmptyGroups: true
```

Comment:
Let's define a top-level constant with default options and merge them here.

```
this.options = Object.assign({}, DEFAULT_OPTIONS, options);
```

This way we can list all options in one place.

BTW, I think avoiding building unnecessary groups makes sense, not sure why we'd cut that. Otherwise the generated scenegraph can be unnecessarily deep and "verbose".

Reply:
I think we will have more edge-cases and will be harder to debug.

Reply:
> I think we will have more edge-cases and will be harder to debug.

Well, just to explain where I am coming from, I noticed is that in many cases we'll just end up wrapping a `Mesh` (or `MeshModel` or ...) in a `Group`. 

```
Group
  | --------|
Group     Group
  |         | 
Mesh       Mesh
```

This looked unnecessary to me, and it seemed to me that simplifying this by dropping (i.e. not generating) the middle layer of groups shouldn't make debugging harder.

My intention was not to do any complex scenegraph transformations but just be a little efficient in avoiding generation of unnecessary nodes.